apps/web/src/app/api/sendMail/route.ts

```diff
@@ -0,0 +1,29 @@
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
This is an API route, As I can understand with this we need to call API to send an email. I suggest you create a service we can use in our backend to send emails.

The correct folder for creating a service is `packages/api/src/lib` where all backend logic exists.

For an example of a service, you can check `apps/web/src/lib/services/conversation.ts` for all AI-related tasks.

There should be atleast two methods in this mail service: `service.sendMaile()` for sending mail and `service.canSendMail()` for verifying if the mail service is configured correctly. The mail service should be a choice; if the mail service is not set up, the app should still function properly.

Reply:
All the task done except this one please elaborate this one to me.

Reply:
Basically, We can create a **function** which can be called to send email (Right now you have create a API route).
Because this function should be secure and **not accessible on the client side**, we will use it in our backend code only (which is inside `packages/api`).

If you are confused about the folder structure, This repo is monorepo using [Turborepo](https://turbo.build/repo).

We can discuss on call if you still have any doubt.

Reply:
Just check this screenshot. 
And also the calling function will be in the `apps/web/src/lib/services/` directory right?
![Screenshot from 2024-06-05 15-37-23](https://github.com/growupanand/ConvoForm/assets/105445488/c9c9f473-119d-4a3c-9eee-1ee59336ac37)

Reply:
As I can see in the screenshot, there are two files: `canSendMail.ts` and `sendMail.ts`. We think we can merge them into one file because both are related to the same service. We try to make one service, one file, with all functions related to that service in the same file.

Currently, we are not calling it anywhere, so there is no need to put anything inside `apps/web/src/lib/services/`.

We can discuss this on Discord's channel if you want.
https://discord.com/channels/1231112132595028008/1238597633522663454

Reply:
![image](https://github.com/growupanand/ConvoForm/assets/105445488/31d48fba-5f66-4b4a-87b1-98acd9d298d4)

Done. Should I make a new commit now??
Also discord link is not working.

Reply:
If everything is done, you can create a new commit.
For the discord link, it looks like some kind of permission issue; let me raise this with the GSSoC team.